src/components/GoalPage/GoalPage.tsx

```diff
@@ -475,6 +522,51 @@ export const GoalPage = ({ user, ssrTime, params: { id } }: ExternalPageProps<{
                         ))}
                     </IssueMeta>
 
+                    {!goal.tags.length ? (
+                        nullable(goal._isEditable, () => (
+                            <IssueMeta title={tr('Tags')}>
+                                <StyledInlineInput>
+                                    <TagComboBox
+                                        disabled={(goal.tags || []).length >= tagsLimit}
+                                        placeholder={tr('Enter tag title')}
+                                        onChange={onTagAdd}
+                                        renderTrigger={(props) => (
+                                            <StyledInlineTrigger
+                                                icon={<IconPlusCircleOutline noWrap size="xs" />}
+                                                text={tr('Add tag')}
+                                                onClick={props.onClick}
+                                            />
+                                        )}
+                                    />
+                                </StyledInlineInput>
+                            </IssueMeta>
+                        ))
+                    ) : (
+                        <>
+                            <IssueMeta title={tr('Tags')}>
+                                {goal.tags?.map((tag) => (
+                                    <Tag key={tag.id}>{tag.title}</Tag>
+                                ))}
+                            </IssueMeta>
+                            {nullable(goal._isEditable, () => (
+                                <StyledInlineInput>
+                                    <TagComboBox
```

Comment:
you render the same combobox in both branches of a ternary. Is there any reason for that?

Reply:
task conditions. I see such a fork in the path